packages/stream-model-handler/src/__tests__/model-handler.test.ts

```diff
@@ -474,8 +474,6 @@ describe('ModelHandler', () => {
 
     // apply anchor
     const anchorProof = {
-      blockNumber: 123456,
-      blockTimestamp: 1615799679,
       chainId: 'fakechain:123',
     }
```

Comment:
These anchor proofs are a bit silly now. ideally they should include a txHash, etc. not super important though.

Reply:
yeah personally I like using the bare minimum required in a unit test.  Makes it clear to readers what's actually being used.  I could add txHash to make it look like a real anchorProof, but then readers might think that's actually relevant to the test, which it isn't